examples/semantics/semantics_model_example.py

```diff
@@ -65,7 +65,7 @@
     #
     # More details about the uniqueness later
 
-    # ## 2.1 Instantiating a class
+    # ## 2.1. Instantiating a class
```

Comment:
why do you put dots here? that is rather unusual when you structure a text into subsections --> have a lokk at latex